core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('Responses Service', function() {
   var interactionData = null;
   var interactionDataWithRules = null;
   var LoggerService = null;
-
+  var oldToNewListMapping = {};
```

Comment:
Shouldn't you have some unit tests where this is not an empty dict, to ensure that it is applied correctly?

Also, make sure to include tests for the null/undefined case (if this can be null/undefined). In general make sure that the tests are in line with the values that are likely to be passed around in the code.

Reply:
Done.